mydocs/README.md

```diff
@@ -1,8 +1,6 @@
-[@togglecorp/fujs](README.md) › [Globals](globals.md)
-
-# @togglecorp/fujs
+@togglecorp/fujs / [Exports](modules.md)
 
 # Fujs
-[![Install Size](https://packagephobia.now.sh/badge?p=@togglecorp/fujs)](https://packagephobia.now.sh/result?p=@togglecorp/fujs@1.9.1) [![Build Status](https://travis-ci.com/toggle-corp/fujs.svg?branch=develop)](https://travis-ci.com/toggle-corp/fujs) [![Maintainability](https://api.codeclimate.com/v1/badges/a48a61764483bcb17971/maintainability)](https://codeclimate.com/github/toggle-corp/fujs/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/a48a61764483bcb17971/test_coverage)](https://codeclimate.com/github/toggle-corp/fujs/test_coverage) [![codecov](https://codecov.io/gh/toggle-corp/fujs/branch/develop/graph/badge.svg)](https://codecov.io/gh/toggle-corp/fujs) [![NPM Package](https://img.shields.io/npm/v/@togglecorp/fujs.svg?style=flat-square)](https://www.npmjs.com/package/@togglecorp/fujs)
+[![Install Size](https://packagephobia.now.sh/badge?p=@togglecorp/fujs)](https://packagephobia.now.sh/result?p=@togglecorp/fujs@1.9.2-beta.0) [![Build Status](https://travis-ci.com/toggle-corp/fujs.svg?branch=develop)](https://travis-ci.com/toggle-corp/fujs) [![Maintainability](https://api.codeclimate.com/v1/badges/a48a61764483bcb17971/maintainability)](https://codeclimate.com/github/toggle-corp/fujs/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/a48a61764483bcb17971/test_coverage)](https://codeclimate.com/github/toggle-corp/fujs/test_coverage) [![codecov](https://codecov.io/gh/toggle-corp/fujs/branch/develop/graph/badge.svg)](https://codecov.io/gh/toggle-corp/fujs) [![NPM Package](https://img.shields.io/npm/v/@togglecorp/fujs.svg?style=flat-square)](https://www.npmjs.com/package/@togglecorp/fujs)
```

Comment:
The docs was generated before the package was published and hence has an outdated version id.